d2bs/kolbot/D2BotLead.dbj

```diff
@@ -392,10 +392,19 @@ MainSwitch:
 
 		break;
 	case 2: // Waiting In Line
-		D2Bot.updateStatus("Waiting...");
-		locationTimeout(StarterConfig.WaitInLineTimeout * 1e3, location);
-		ControlAction.click(6, 433, 433, 96, 32);
-
+		let positionInLine = ControlAction.getQueueTime();
```

Comment:
most of this is redundant
```
let line = 0;
let now;
while ((now = ControlAction.getQueueTime())) {
	if (line != now) {
		D2Bot.updateStatus("Position in line " + positionInLine);
		line = now;
	}
	delay(1000);
}
```

Reply:
Also consider keeping the "default" waiting in line delay, for example, if you are here and you read 0 for some reason (if that's possible) you may want some sort of delay to allow UI to catch up